elements/dash-tab-panel/dash-tab-panel.e2etest.js

```diff
@@ -0,0 +1,28 @@
+const helper = require('../../tools/selenium-helper.js');
+const expect = require('chai').expect;
+const Key = require('selenium-webdriver').Key;
+
+describe('dash-tab-panel', function() {
+  beforeEach(function() {
+    return this.driver.get(`${this.address}/dash-tab-panel_demo.html`)
+      .then(_ => helper.waitForElement(this.driver, 'dash-tab-panel'));
+  });
+
+  it('should focus the next element on [arrow right]', async function() {
+    const found =
+      await helper.pressKeyUntil(
+        this.driver,
+        Key.TAB,
+        `return document.activeElement.getAttribute('aria-role') === 'tab';`
+      );
+    expect(found).to.equal(true);
+
+    await this.driver
+      .executeScript(`window.firstTab = document.activeElement;`);
+    await this.driver.actions().sendKeys(Key.ARROW_RIGHT).perform();
+    const rightFocus = await this.driver.executeScript(`
+      return window.firstTab.nextElementSibling === document.activeElement;
```

Comment:
What do you think about looking for the next element where `aria-role === 'tab'` instead of just looking for the next focusable sibling?